### ISSUE TITLE
Doco: ELI5 how to get going with Docker image

### ISSUE THREAD

**Issue by PhilHudson** [200ok-ch/organice#785]
Before creating a new issue, please read the "How we work with issues" section in the documentation: https://organice.200ok.ch/documentation.html#how_we_work_with_issues

**Is your feature request related to a problem? Please describe.**
A clear and concise description of the process for pulling and configuring the docker image to work with my org files located in (say) ~/org/. Starting from nothing but docker.io already installed and ending with my agenda (or just my main ToDo.org) showing in a web browser.

**Describe the solution you'd like**
Add suitable step-by-step instructions to the Readme.

**Describe alternatives you've considered**
Dissecting the docker image file and hoping I've understood everything.

**Additional context**

**Comment by schoettl** [comment]
Hi Phil,

Would you mind cleaning up your issue a bit, i.e. removing placeholder text? It would be easier for maintainers to understand would you want.

What's the problem with https://github.com/200ok-ch/organice#with-docker-compose ? And have you also seen https://github.com/200ok-ch/organice#webdav ?

**Comment by PhilHudson** [comment]
Please document how to pull and configure the docker image to work with a user's org files located in (say) ~/org/. Starting from nothing but docker.io already installed and ending with my agenda (or just my main ToDo.org) showing in a web browser.

**Comment by munen** [comment]
Thank you for your request.

The documentation of organice is not meant to be a replacement of documentation of Docker and command-line tooling. Having said so, we have documentation that is sufficient for people comfortable with these tools: https://organice.200ok.ch/documentation.html#docker

If you're not comfortable using Docker and the command line, already, we have you covered: Use the instance of organice that we're already hosting for you: https://organice.200ok.ch

If you wish to add to the documentation, feel free to create a PR.
